.github/workflows/comments.yml

```diff
@@ -26,32 +20,31 @@ jobs:
         uses: xom9ikk/dotenv@v1.0.2
         with:
           path: .github/workflows/
+      - name: Move into From Bitnami
+        uses: peter-evans/create-or-update-project-card@v2
+        if: ${{ github.actor == 'bitnami-bot' }}
+        with:
+          project-name: Support
+          column-name: From Bitnami
+          token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
       - name: Move into Pending
         uses: peter-evans/create-or-update-project-card@v2
-        if: ${{ (!contains(github.event.issue.labels.*.name, 'bitnami')) && contains(fromJson(env.BITNAMI_TEAM), github.event.comment.user.login) }}
+        if: ${{ github.actor != 'bitnami-bot' && (!contains(github.event.issue.labels.*.name, 'bitnami')) && contains(fromJson(env.BITNAMI_TEAM), github.actor) }}
         with:
           project-name: Support
           column-name: Pending
-          token: "${{ secrets.GITHUB_TOKEN }}"
+          token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
       - name: Move into In Progress
         uses: peter-evans/create-or-update-project-card@v2
-        if: ${{ contains(github.event.issue.labels.*.name, 'in-progress') && (!contains(fromJson(env.BITNAMI_TEAM), github.event.comment.user.login)) }}
+        if: ${{ github.actor != 'bitnami-bot' && contains(github.event.issue.labels.*.name, 'in-progress') && (!contains(fromJson(env.BITNAMI_TEAM), github.actor)) }}
         with:
           project-name: Support
           column-name: In progress
-          token: "${{ secrets.GITHUB_TOKEN }}"
+          token: "${{ secrets.BITNAMI_BOT_TOKEN }}"
```

Comment:
When you move into In Progress, do you really need to trigger the `moving-cards.yml` workflow?